ec2/spark_ec2.py

```diff
@@ -569,6 +569,9 @@ def launch_cluster(conn, opts, cluster_name):
         master_nodes = master_res.instances
         print "Launched master in %s, regid = %s" % (zone, master_res.id)
 
+    # The insert of waiting time corresponds to issue [SPARK-4983]
+    print "Waiting for AWS to propagate the information of instances..."
```

Comment:
Nit: Please rephrase this to "Waiting for AWS to propagate instance metadata..."